shared/haml/hamburger.haml

```diff
@@ -124,6 +152,8 @@
       - if level || script_level
         %li
           %a{href: CDO.code_org_url('/educate/applab')}=I18n.t("#{loc_prefix}tutorials")
+      -if language == "en"
```

Comment:
Again, it would be neat to use a _show_ variable here rather than use implicit knowledge of what appears for an en user.